cartography/intel/gcp/cloudrun.py

```diff
@@ -383,6 +416,18 @@ def get_cloudrun_services(cloudrun: Resource, project_id: str, common_job_parame
         else:
             raise
 
+@timeit
+def transform_services(services: List[Dict], project_id: str) -> List[Dict]:
+    services = []
+    for item in services:
+        item['region'] = item.get('metadata').get('labels').get('cloud.googleapis.com/location')
+        item['consolelink'] = gcp_console_link.get_console_link(project_id=project_id,\
+                cloud_run_service_name=item.get('metadata').get('name'), resource_name='cloud_run_revision')
```

Comment:
Since these are cloud run services, should the resource_name be `cloud_run_revision`?

Reply:
use resource_name `cloud_run_service` here